modules/swagger-jaxrs2/src/main/java/io/swagger/v3/jaxrs2/Reader.java

```diff
@@ -664,6 +660,20 @@ public OpenAPI read(Class<?> cls,
         return openAPI;
     }
 
+    protected void decorateOperation(Class<?> cls, Operation operation, Method method) {
+        final Iterator<OpenAPIExtension> chain = OpenAPIExtensions.chain();
+        if (chain.hasNext()) {
+            final OpenAPIExtension extension = chain.next();
+            extension.decorateOperation(operation, method, chain);
+        }
+
+        final Iterator<OpenAPIExtension> chain2 = OpenAPIExtensions.chain();
+        if (chain2.hasNext()) {
+            final OpenAPIExtension extension = chain2.next();
+            extension.decorateOperation(cls, operation, method, chain2);
+        }
+    }
+
```

Comment:
I would rather use same iterator and invoke in sequence the 2 overloads, is there a reason why two iterations are necessary?